sdl_viewer/src/main.rs

```diff
@@ -38,6 +40,17 @@ use std::str;
 const FRAGMENT_SHADER: &'static str = include_str!("../shaders/points.fs");
 const VERTEX_SHADER: &'static str = include_str!("../shaders/points.vs");
 
+fn reshuffle(new_order: &[usize], old_data: Vec<u8>, bytes_per_point: usize) -> Vec<u8> {
```

Comment:
`point` -> `vertex`? I think that is more common GL term for the data at this point in time, i.e. before rendering them as points.